packages/agents/router/test/lib/operations/execute.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { constants, utils, BigNumber } from "ethers";
 import { reset, restore } from "sinon";
-import { Bid, DEFAULT_ROUTER_FEE, expect, OriginTransfer } from "@connext/nxtp-utils";
+import { Bid, expect, OriginTransfer } from "@connext/nxtp-utils";
```

Comment:
in line with the previous comment, pls remove `DEFAULT_ROUTER_FEE` from codebase, neither are needed